src/uct/tcp/tcp_ep.c

```diff
@@ -614,11 +612,11 @@ static void uct_tcp_iface_check_rp_filter(uct_tcp_iface_t *iface)
         return;
     }
 
-    if ((rp_filter != 0) && (rp_filter != 2)) {
-        ucs_diag("tcp_iface %p: rp_filter of %s is set to %ld, "
-                 "for stable work it should be 0 or 2, see "
+    if (rp_filter == 1) {
+        ucs_diag("tcp_iface %p: rp_filter of %s is set to strict mode, "
+                 "it may cause packet check failures, see "
```

Comment:
what is "packet check failure"? how does it show on UCX level?

Reply:
AFAIU packets will be dropped on TCP level, so UCT should fail by timeout. I don't this I can reproduce this scenario since according to the documentation it depends on topology and routing configuration.

Reply:
1. "tcp_iface %p: net.ipv4.conf.%s.rp_filter is set to strict mode (1), connections may fail"
2. if iface binding is optional (not YES), shall we skip it when rp_filter==1?